prow/create-secrets-for-workload-cluster.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Source development/pre-install-prow.sh
+
+set -o errexit
+
+if [ -z "$PROJECT" ]; then
+      echo "\$PROJECT is empty"
+      exit 1
+fi
+
+if [ -z "$GCLOUD_COMPUTE_ZONE" ]; then
+      echo "\$GCLOUD_COMPUTE_ZONE is empty"
+      exit 1
+fi
+
+if [ -z "$WORKLOAD_CLUSTER_NAME" ]; then
+      echo "\$WORKLOAD_CLUSTER_NAME is empty"
+      exit 1
+fi
+
+if [ -z "$GOPATH" ]; then
+      echo "\$GOPATH is not set"
+      exit 1
+fi
+
+### Cloning k8s.io/test-infra is a prerequisite here
+git clone "git@github.com:kubernetes/test-infra.git" "${GOPATH}/src/k8s.io/test-infra" || cd "${GOPATH}/src/k8s.io/test-infra/prow"
+
+### Reference: https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkbuild-cluster#usage
+bazel run //prow/cmd/mkbuild-cluster -- \
```

Comment:
why do we require bazel? Maybe it is enough to use go run? For example I run mkpj as a go application. If bazel is really required, then we should have it documented and checked at the beginning of the script.

Reply:
Changed to `go run`